From 742f20fcc31065fb9b35e40a1815dd91ea1148e4 Mon Sep 17 00:00:00 2001
From: Sergii Pylypenko <x.pelya.x@gmail.com>
Date: Tue, 3 Jun 2014 18:44:34 +0300
Subject: [PATCH 0134/1053] Revert "Erase queued command also when depot
 direction/airport type/station type changes."

This reverts commit 60a7ac9f3c1d4f271f4392859d68df875d612fe7.
---
 src/airport_gui.cpp | 2 --
 src/dock_gui.cpp    | 1 -
 src/rail_gui.cpp    | 3 ---
 src/road_gui.cpp    | 2 --
 4 files changed, 8 deletions(-)

diff --git a/src/airport_gui.cpp b/src/airport_gui.cpp
index 8b49b998ce..aa57562241 100644
--- a/src/airport_gui.cpp
+++ b/src/airport_gui.cpp
@@ -423,8 +423,6 @@ public:
 
 	void UpdateSelectSize()
 	{
-		EraseQueuedTouchCommand();
-
 		if (_selected_airport_index == -1) {
 			SetTileSelectSize(1, 1);
 			this->DisableWidget(WID_AP_LAYOUT_DECREASE);
diff --git a/src/dock_gui.cpp b/src/dock_gui.cpp
index 4e6ed21402..1d2b2e8abc 100644
--- a/src/dock_gui.cpp
+++ b/src/dock_gui.cpp
@@ -475,7 +475,6 @@ public:
 				_settings_client.gui.station_show_coverage = (widget != BDSW_LT_OFF);
 				this->LowerWidget(_settings_client.gui.station_show_coverage + BDSW_LT_OFF);
 				if (_settings_client.sound.click_beep) SndPlayFx(SND_15_BEEP);
-				EraseQueuedTouchCommand();
 				this->SetDirty();
 				break;
 		}
diff --git a/src/rail_gui.cpp b/src/rail_gui.cpp
index 05b8bc4e7e..7071ccc849 100644
--- a/src/rail_gui.cpp
+++ b/src/rail_gui.cpp
@@ -1367,8 +1367,6 @@ public:
 				break;
 			}
 		}
-
-		EraseQueuedTouchCommand();
 	}
 
 	virtual void OnTick()
@@ -1723,7 +1721,6 @@ struct BuildRailDepotWindow : public PickerWindowBase {
 				_build_depot_direction = (DiagDirection)(widget - WID_BRAD_DEPOT_NE);
 				this->LowerWidget(_build_depot_direction + WID_BRAD_DEPOT_NE);
 				if (_settings_client.sound.click_beep) SndPlayFx(SND_15_BEEP);
-				EraseQueuedTouchCommand();
 				this->SetDirty();
 				break;
 		}
diff --git a/src/road_gui.cpp b/src/road_gui.cpp
index 341d15cbfc..890eafed48 100644
--- a/src/road_gui.cpp
+++ b/src/road_gui.cpp
@@ -882,7 +882,6 @@ struct BuildRoadDepotWindow : public PickerWindowBase {
 				_road_depot_orientation = (DiagDirection)(widget - WID_BROD_DEPOT_NE);
 				this->LowerWidget(_road_depot_orientation + WID_BROD_DEPOT_NE);
 				if (_settings_client.sound.click_beep) SndPlayFx(SND_15_BEEP);
-				EraseQueuedTouchCommand();
 				this->SetDirty();
 				break;
 
@@ -1029,7 +1028,6 @@ struct BuildRoadStationWindow : public PickerWindowBase {
 				_settings_client.gui.station_show_coverage = (widget != WID_BROS_LT_OFF);
 				this->LowerWidget(_settings_client.gui.station_show_coverage + WID_BROS_LT_OFF);
 				if (_settings_client.sound.click_beep) SndPlayFx(SND_15_BEEP);
-				EraseQueuedTouchCommand();
 				this->SetDirty();
 				break;
 
-- 
2.55.0

